src/components/pages/gfcrPages/GfcrIndicatorSetForm/subPages/F4Form.js

```diff
@@ -217,12 +235,19 @@ const F4Form = ({
           type="number"
           unit="kg/ha"
           {...formik.getFieldProps('f4_3')}
-          onBlur={(event) => handleInputBlur(formik, event, 'f4_3', true)}
```

Comment:
sometimes calling this function was done with a fourth unnamed parameter. What was hidden there is that passing true as the 4th param was reformatting the input to only have 1 decimal. I opted to use the existing decimal restriction function to stay consistent in code and behaviour, so now onBlur only deals with resetting defaults. (the new code wont allow the user to type too many decimals, the old way just overwrote whatever they input to have less decimals after they clicked away so this is more consistent in terms of UX too now)